pulumi/index.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const aws = require('@pulumi/aws');
+const pulumi = require('@pulumi/pulumi');
+
+const config = new pulumi.Config();
+
+// Create an AWS resource (S3 Bucket)
+const bucket = new aws.s3.Bucket('tailor');
+
+// Create new IAM user
+const user = new aws.iam.User('tailor-user', {
+  name: 'tailor'
+});
+// Create access and secret keys for the user
+const userCredentials = new aws.iam.AccessKey('tailor-user-key', {
+  user: user.name
+});
+
+const policy = new aws.iam.Policy('tailor-bucket-access', {
+  description: 'Allow tailor user full control over the tailor S3 bucket',
+  policy: {
+    Version: '2012-10-17',
+    Statement: [{
+      Sid: 'AllowRetouchS3Management',
+      Effect: 'Allow',
+      Resource: bucket.arn.apply(arn => arn + '/*'),
+      Action: 's3:*'
+    }]
+  }
+});
+
+// Attack policy to the user
+// eslint-disable-next-line no-new
+new aws.iam.UserPolicyAttachment('user-policy-attachment', {
+  user: user.name,
+  policyArn: policy.arn
+});
+
+// Create postgres db instance
+const db = new aws.rds.Instance('postgresdb', {
+  engine: 'postgres',
+  instanceClass: 'db.t2.micro',
+  allocatedStorage: 20,
+  name: 'tailor', // db name
+  username: 'tailor',
+  password: 'tailor123',
```

Comment:
Can we get this from dotenv?

Reply:
Of course, this was the initial phase. It's not ready yet for review 😅 😄

Reply:
No worries I made sort of _todo comment_ there  😉
Blame @kronicker for my snooping 😀